core-plugins/src/test/java/co/cask/hydrator/plugin/batch/ETLMapReduceTestRun.java

```diff
@@ -300,6 +300,133 @@ public void testTableToTableWithValidations() throws Exception {
     }
   }
 
+
+  @SuppressWarnings("ConstantConditions")
+  @Test
+  public void testInvalidTableConfigurationComplexType() throws Exception {
```

Comment:
These tests are fairly heavyweight and slow since they require hitting handlers and messing with jars, etc.

Since we're just testing schema validation, we should be able to write unit tests for just the sink. Like a TableSinkTest that creates the sink directly, sets the input schema, and calls configurePipeline directly.  That type of test would be much more lightweight and still test the same thing